CIP-0142/README.md

```diff
@@ -0,0 +1,115 @@
+---
+CIP: 0142
```

Comment:
```suggestion
CIP: 142
```
also please trim the leading `0` from here, as for all our other CIPs (I'd included that in https://github.com/cardano-foundation/CIPs/pull/972#pullrequestreview-2653578001 but somehow GitHub ate it)